topics/metagenomics/tutorials/metatranscriptomics/slides.html

```diff
@@ -529,19 +562,27 @@
 
 .image-75[![results](../../images/metatranscriptomics/results_cellulose_degradation.png)]
 
+???
+explain about datasets first
+cellulose 1,4 beta-cellulobiosidase-> hydrolysis of cellulose
+upregulation
 ---
 
 ## Functions associated with a selected taxon
 
 .image-75[![results](../../images/metatranscriptomics/results_functions_taxon.png)]
 
+???
+gene abundance-coprothermobacter and clostridium. here we see the glycolysis pathway showing same trend
```

Comment:
@subinamehta can you put a little bit more effort here, to make those proper sentences and such? Thanks!

Reply:
OK..I will add the explanations 👍